admin/class-bulk-editor-list-table.php

```diff
@@ -152,13 +152,14 @@ function display_tablenav( $which ) {
 
 			<?php if ( 'top' === $which ) { ?>
 			<form id="posts-filter" action="" method="get">
-				<input type="hidden" name="page" value="wpseo_bulk-editor" />
-				<input type="hidden" name="type" value="<?php echo esc_attr( $this->page_type ); ?>" />
-				<input type="hidden" name="orderby" value="<?php echo esc_attr( $_GET['orderby'] ); ?>" />
-				<input type="hidden" name="order" value="<?php echo esc_attr( $_GET['order'] ); ?>" />
-				<input type="hidden" name="post_type_filter" value="<?php echo esc_attr( $_GET['post_type_filter'] ); ?>" />
+				<input type="hidden" name="page" value="wpseo_bulk-editor"/>
+				<input type="hidden" name="type" value="<?php echo esc_attr( $this->page_type ); ?>"/>
+				<input type="hidden" name="orderby" value="<?php echo esc_attr( $_GET['orderby'] ); ?>"/>
+				<input type="hidden" name="order" value="<?php echo esc_attr( $_GET['order'] ); ?>"/>
+				<input type="hidden" name="post_type_filter"
+				       value="<?php echo esc_attr( $_GET['post_type_filter'] ); ?>"/>
```

Comment:
code style thing in PHPStorm I think, will have to look it up, annoys me too. Will fix.